src/lib/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkHeader.tsx

```diff
@@ -33,40 +30,25 @@ const MyCollectionArtworkHeader: React.FC<MyCollectionArtworkHeaderProps> = (pro
     return isProcessing
   }
 
-  const hasImagesStillProcessing = (mainImage: any, imagesToCheck: MyCollectionArtworkHeader_artwork["images"]) => {
-    if (!isImage(mainImage) || imageIsProcessing(mainImage)) {
-      return true
-    }
-
-    if (!imagesToCheck) {
-      return false
-    }
-
-    const concreteImages = imagesToCheck as Image[]
-    const stillProcessing = concreteImages.some((image) => imageIsProcessing(image))
-    return stillProcessing
-  }
-
   const renderMainImageView = () => {
     if (!isImage(defaultImage) || imageIsProcessing(defaultImage)) {
       return (
-        <Flex
-          style={{ height: 300, alignItems: "center", justifyContent: "center", backgroundColor: color("black10") }}
-        >
+        <Flex style={{ height: 300, alignItems: "center", justifyContent: "center", backgroundColor: color("black5") }}>
           <ArtworkIcon style={{ opacity: 0.6 }} height={100} width={100} />
           <Text style={{ opacity: 0.6 }}>
             {images && images?.length > 0 ? "Processing photos" : "Processing photo"}
           </Text>
         </Flex>
       )
     } else {
-      // TODO: figure out if "normalized" is the correct version
+      const screenDimensions = Dimensions.get("screen")
```

Comment:
```suggestion
      const screenDimensions = useScreenDimensions()
```

for iPad, which allows rotation